app/server.go

```diff
@@ -371,6 +376,7 @@ func NewServer(options ...Option) (*Server, error) {
 	}
 
 	serviceMap := map[ServiceKey]interface{}{
+		ChannelKey:   channelWrapper,
```

Comment:
Interesting. For Channels we are passing this during init, and for posts, we are attaching it in the channels initializer. Should we try to make both happen at the same time?

Reply:
Alright